test/e2e/utils/record-helpers.js

```diff
@@ -284,9 +307,9 @@ exports.testPresentation = function (tableParams) {
             // empty related table should show
             expect(chaisePage.recordPage.getRelatedTable(tableDisplayname).isPresent()).toBeTruthy();
             //check the no results text appears properly
-            return chaisePage.recordPage.getNoResultsRow().getText();
-        }).then(function(text) {
-            expect(text).toBe(noResultsMessage);
+            return chaisePage.recordPage.getNoResultsRow();//.getText();
+        }).then(function(emptyTab) {
+            expect(emptyTab[1].getText()).toBe(noResultsMessage);
```

Comment:
This is generally a better way for checking text values in the UI. `expect()` will actually evaluate the promise and use the result of that promise for the comparison.